### DISCUSSION TITLE
Keyboard backlight modes

### DISCUSSION THREAD

**Discussion by hitmefast** [seerge/g-helper#911] [General]
Is there any custom settings or way to make the keyboard mode react when to start typing?
Let me explain.. If I use breathing mode in some of my old keyboards or almost any mode I as soon as I type it will bright more than when its not in use, and this case is just kind of annoying, since at night as soon as I start typing and check the keyboard its in the 'breath' part and its not showing any key and I need to wait for it. Or a simillar case is when I want to start typing and I cant see the keyboard and then a second later it is bright. Is there any way to set a min value to brightness? or perhaps a start typing detection mode? or waterfall/matrix mode? I know might be a niche but the keyboard is almost 80% or more of what people use and see of the keyboard and therefore has so much impact in both usability and appeal. :)

**Comment by seerge** [comment]
There is already backlight timeout settings that do what you want. It will turn off backlight after set amount of seconds, when laptop is not in use, and turn back on as soon as you start typing.
